utils/build/docker/python/django/app/urls.py

```diff
@@ -652,6 +652,65 @@ def view_iast_source_path(request):
     return HttpResponse("OK")
 
 
+@csrf_exempt
+def view_iast_sampling_by_route_method(request, id):
```

Comment:
Can you add a short docstring explaining why the multi executes

Reply:
Pull Request updated!